Misc/NEWS.d/next/Library/2024-10-24-10-49-47.gh-issue-124452.eqTRgx.rst

```diff
@@ -0,0 +1 @@
+Fix email parsing bug that introduced spurious leading whitespace into header values when the header includes a newline character after the ';' and before the value.

```

Comment:
Maybe you should used rst label.
```
:mod:`email`
```
For more to see https://devguide.python.org/documentation/markup/

Reply:
true, I'll have look at this doc